### PR TITLE
must-gather: add dmesg data to must-gather

### PR DESCRIPTION
**Pull request by marioferh**
Signed-off-by: Mario Fernandez <mariofer@redhat.com>

### CODE REVIEW

**Comment by marioferh** [comment]
First approach. Open to discuss re-implementation of dmesg.

Should we take the data directly of syslogd or klogd?

File: must-gather/collection-scripts/gather_nodes [hunk 1]
```diff
@@ -62,6 +62,7 @@ do
     oc exec $pod -n perf-node-gather -- lspci -nvv > $NODE_PATH/lspci
     oc exec $pod -n perf-node-gather -- lscpu -e > $NODE_PATH/lscpu
     oc exec $pod -n perf-node-gather -- cat /proc/cmdline > $NODE_PATH/proc_cmdline
+    oc exec $pod -n perf-node-gather -- dmesg $NODE_PATH/dmesg.json
```
Comment (fromanirh) [inline must-gather/collection-scripts/gather_nodes:65]: missing shell redirect. Also, the output is not JSON

**Review by fromanirh — COMMENTED** [review]
what's the rationale for which addition? in which cases we anticipate this data to be useful?

File: must-gather/collection-scripts/gather_nodes (hunk 1 above)
Comment (marioferh) [reply must-gather/collection-scripts/gather_nodes]: fast push, fixed

**Comment by fromanirh** [comment]
/lgtm
/hold
in the long term, we should get rid of all the external deps and only user `gather_sysinfo`. For the time being, better to have this info now

**Comment by fromanirh** [comment]
any chance to fix these flakes somehow?

**Comment by marioferh** [comment]
> any chance to fix these flakes somehow?

`[INFO] Deploying performance operator and profile.
[WARN] Deployment did not fully succeed yet, retrying in 10 sec, 1 retries left
[INFO] Deploying performance operator and profile.
machineconfigpool.machineconfiguration.openshift.io/worker-cnf unchanged
Warning: v1alpha1 is deprecated and should be removed in the next release, use v2 instead
Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "vwb.performance.openshift.io": failed to call webhook: Post "[https://performance-operator-service.openshift-performance-addon-operator.svc:443/validate-performance-openshift-io-v2-performanceprofile?timeout=10s](https://performance-operator-service.openshift-performance-addon-operator.svc/validate-performance-openshift-io-v2-performanceprofile?timeout=10s)": no endpoints available for service "performance-operator-service"
[WARN] At least one deployment failed, giving up
[ERROR] Deployment failed, giving up.
make[1]: *** [Makefile:233: cluster-deploy] Error 1
make[1]: Leaving directory '/go/src/github.com/openshift-kni/performance-addon-operators'
make: *** [Makefile:272: operator-upgrade-tests] Error 2`

This one the deploying is failing, seems as a infra issue?

**Pull request merged**